src/pages/PlaylistPage.js

```diff
@@ -1,40 +1,138 @@
 import React, {PropTypes} from 'react';
-import NavLink from '../components/Navigation/NavLink';
+import {connect} from 'react-redux';
+import LessonFilter from '../components/Filter/LessonFilter';
+import LessonList from '../components/LessonList/LessonList';
+import LevelNavigation from '../components/LessonList/LevelNavigation';
+import PlaylistNavigation from '../components/Playlist/PlaylistNavigation';
+import {getFilteredCourses} from '../selectors/course';
+import * as actionCreators from '../action_creators';
+import {capitalize} from '../util';
+import {
+  Button,
+  Col,
+  Collapse,
+  Grid,
+  Row
+} from 'react-bootstrap';
 
+export const PlaylistPage = React.createClass({
+  getInitialState() {
+    return {
+      showMobileFilter: false,
+      showMobilePlaylists: false,
+      showMobileLevelNavigation: false
```

Comment:
Vel, det er forskjellige filosofier her. Noen mener at all state skal gå i state-treet, bl.a. slik at man kan bruke React stateless functional components (https://facebook.github.io/react/docs/reusable-components.html#stateless-functions), og også lagre state-treet i localstore slik at når en bruker åpner siden igjen så er alt helt likt. Og dersom man skal teste components, så er det lettere, siden components ikke har state.

På den annen side, så lenge man bruker React.createClass og ikke er interessert i lagre brukerens siste actions, men begynner på "nytt" når man reloader siden, så er det ikke så farlig. Uansett ikke så vanskelig å flytte inn i state-treet senere.

HVIS vi skulle flytte det inn, så kunne man ha en egen UI-del i state-treet, og bare lagt alt der.

Forøvrig en del diskusjon om dette på nettet. Til og med om man kunne implementert noe der "setState" automatisk lagret til state-treet.